### PR TITLE
refactor(cli): move dataset formatting to _format

### PR DESCRIPTION
**Pull request by jirikuncar**
just a cleanup

### CODE REVIEW

File: tests/cli/test_datasets.py [hunk 1]
```diff
@@ -22,8 +22,11 @@
 import os
 
 import git
+import pytest
 
 from renku import cli
+from renku._compat import Path
+from renku.cli._format.datasets import FORMATS
```
Comment (jsam) [inline tests/cli/test_datasets.py]:
```suggestion
from renku.cli._format.datasets import FORMATS as DATASETS_FORMATS
```

File: tests/cli/test_datasets.py [hunk 2]
```diff
@@ -69,33 +72,24 @@ def test_datasets_import(data_file, data_repository, runner, project, client):
     assert result.exit_code == 0
 
 
-def test_datasets_list_empty(runner, project):
+@pytest.mark.parametrize('output_format', FORMATS.keys())
```
Comment (jsam) [inline tests/cli/test_datasets.py]:
```suggestion
@pytest.mark.parametrize('output_format', DATASETS_FORMATS.keys())
```

File: tests/cli/test_datasets.py [hunk 3]
```diff
@@ -69,33 +72,24 @@ def test_datasets_import(data_file, data_repository, runner, project, client):
...
 
-    output = result.output.split('\n')
-    assert output.pop(0).split() == ['ID', 'NAME', 'CREATED', 'AUTHORS']
-    assert set(output.pop(0)) == {'-', ' '}
-    assert output.pop(0) == ''
-    assert not output
 
-
-def test_datasets_list_non_empty(runner, project):
+@pytest.mark.parametrize('output_format', FORMATS.keys())
```
Comment (jsam) [inline tests/cli/test_datasets.py]:
```suggestion
@pytest.mark.parametrize('output_format', DATASETS_FORMATS.keys())
```

File: tests/cli/test_datasets.py [hunk 4]
```diff
@@ -145,7 +139,7 @@ def test_repository_file_to_dataset(runner, project, client):
 
     with client.with_dataset('dataset') as dataset:
         assert dataset.name == 'dataset'
-        assert '../../../a' in dataset.files
+        assert Path('../../../a') in dataset.files
```
Comment (jsam) [inline tests/cli/test_datasets.py]: Maybe we should think of adding a check that the file/link actually exists in its location on the filesystem, and not only inside metadata.

File: tests/cli/test_datasets.py (hunk 1 above)
Comment (jirikuncar) [reply tests/cli/test_datasets.py]: not necessary at the moment

File: tests/cli/test_datasets.py (hunk 4 above)
Comment (jirikuncar) [reply tests/cli/test_datasets.py]: no in the scope of this PR

Comment (jsam) [reply tests/cli/test_datasets.py]: And where is scope defined? 😁

Comment (jirikuncar) [reply tests/cli/test_datasets.py]: title

Comment (jsam) [reply tests/cli/test_datasets.py]: 😂 That's not definition. But 👌

**Pull request merged**